### DISCUSSION TITLE
Request add new site specific CSS for https://twitter.com/home

### DISCUSSION THREAD

**Discussion by vegetax6** [W4775/Widify#8] [Ideas]
Request add new site specific CSS for https://twitter.com/home

**Comment by W4775** [comment]
Here's what I came up with: 
![image](https://user-images.githubusercontent.com/31900031/221721539-501dc900-c419-4261-96d2-e57cce203300.png)

I don't really use twitter but I think this should work. I'll add this to the batch of changes.
